templates/appliance/lxd/index.md

```diff
@@ -0,0 +1,85 @@
+---
+wrapper_template: "appliance/shared/_base_appliance_index.html"
+context:
+  name: "LXD"
+  short_name: "lxd"
+  appliance_name: "LXD"
+  company_name: "Canonical"
+  logo: "https://assets.ubuntu.com/v1/3a24a2f9-lxd.png"
+  category: "Developer"
+  meta_description: "With LXD you can run hundreds of containers of a variety of Linux distributions, apply resource limits, pass in directories, USB devices or GPUs and setup any network and storage you want."
+  downloads:
+    raspberrypi: True
+    pc: False
+    intelnuc: True
+  pi:
+    2: False
+    3: False
+    4: True
+  screenshots:
+    1: https://dashboard.snapcraft.io/site_media/appmedia/2018/10/Screenshot_from_2018-10-26_12-57-24.png
+    2: https://dashboard.snapcraft.io/site_media/appmedia/2018/10/Screenshot_from_2018-10-26_14-20-14.png
+    3: https://dashboard.snapcraft.io/site_media/appmedia/2018/10/Screenshot_from_2018-10-26_14-21-43.png
+  links:
+    1:
+      title: "Documentation"
+      url: "https://lxd.readthedocs.io"
+    2:
+      title: "LXD website"
+      url: "https://linuxcontainers.org/"
+    3:
+      title: "License - Apache-2.0"
+      url: "https://www.apache.org/licenses/LICENSE-2.0"
+  compliance:
+    1: "EU GDPR"
+    2: "California law"
+  base: "core18"
+  published_date: "2020-09-18"
+  maintenance_date: "2030-09-18"
+  snaps:
+    1:
+      name: "LXD"
+      icon: "https://assets.ubuntu.com/v1/3a24a2f9-lxd.png"
+      link: "https://snapcraft.io/lxd"
+      publisher: "Canonical"
+      channel: "latest/stable"
+      version: "4.5"
+      published_date: "18 September 2020"
+---
+
+#### System container manager and API
+
+LXD is a system container manager
```

Comment:
It looks like these short sentences without full stops should be headings?

Reply:
fixed